src/gtest/test_foundersreward.cpp

```diff
@@ -0,0 +1,173 @@
+#include <gtest/gtest.h>
+
+#include "main.h"
+#include "utilmoneystr.h"
+#include "chainparams.h"
+#include "utilstrencodings.h"
+#include "zcash/Address.hpp"
+#include "wallet/wallet.h"
+#include "amount.h"
+#include <memory>
+#include <string>
+#include <set>
+#include <vector>
+#include <boost/filesystem.hpp>
+
+// To run tests:
+// ./zcash-gtest --gtest_filter="founders_reward_test.*"
+
+//
+// Enable this test to generate and print 48 testnet 2-of-3 multisig addresses.
+// The output can be copied into chainparams.cpp.
+//
+#if 0
+TEST(founders_reward_test, create_testnet_2of3multisig) {
```

Comment:
cc @daira : Good point.  Pushed commit with new testnet3 addresses and have retained the wallet.dat file so we can use it for testing.